server/build/gitlab-dc.common.yml

```diff
@@ -61,18 +61,22 @@ services:
       timeout: 15s
       retries: 12
   keycloak:
-    image: mattermostdevelopment/mirrored-keycloak:10.0.2
+    image: quay.io/keycloak/keycloak:23.0.7
```

Comment:
@mvitale1989 I noticed this late. Is there a plan to host this image in our `mattermostdevelopment` dockerhub?

Reply:
The medium term plan would be to completely remove the `gitlab-dc.common.yml` file, in favour of using the `docker-compose.yml` file, see [this ticket](https://mattermost.atlassian.net/browse/CLD-8147), so it's probably not worth mirroring this keycloak image for now.
But it's worth understanding if we won't get into any Docker Hub limits as part of this migration, i'll add a note to the ticket